### ISSUE TITLE
memory info not show with poor translated env

### ISSUE THREAD

**Issue by mckaygerhard** [lpereira/hardinfo#31]
in the resumen module in computer section memory info its not show if hardinfo are used in a poorly translated language.

as @lpereira said:

> If it's not updating, it's a bug somewhere else, likely in the shell rather than the computer module. I find it very unlikely, as that is some well-tested part of the code and it works on every machine I've tried HardInfo on.

added 

> it's possible that it's been poorly translated and the UpdateInterval$Memory=1000 has not been translated to match the Memory key.

then i tested as and have right in part, using different lang this king of memory info that show are broken, i tested in russian and in spanish and does not show nothing... 

using this for launch:

```
export LANG=C;export LANGUAGE=C;export GDM_LANG=C;export LC_ALL=C;hardinfo
```

hardinfo correctly show so i'll open a issue and let this pull request opened until u decided what are best to do

**Comment by mckaygerhard** [comment]
firts of all i'll revise the translation po fiels and tried to complete most possible and then post feedback here

**Comment by mckaygerhard** [comment]
related to pull request #28

**Comment by mckaygerhard** [comment]
i research and found its well translated the es.po file:

i have strings:

"UpdateInterval$Memory=1000\n"
and
"UpdateInterval$Memoria=1000\n"

so i dont knwon what other .. i research more in few days..

**Comment by mckaygerhard** [comment]
i confirmed in the isue comment that the pool request mention, see #28, the pull request its not elegant but works for now, quoted :

> Look at the Memory, it can't be translated, because the value disappears with it. There is the comment "//Load Average is not updated if locale is not C, switch locale to C" in /modules/computer.c (https://github.com/lpereira/hardinfo/blob/master/modules/computer.c).

**Comment by mckaygerhard** [comment]
the commits from pull #86 solves other issues like  #38 and this! see picture: i can close when the pull are merged!

![hardinfomemetranslatable](https://user-images.githubusercontent.com/1571724/28135497-6186c4d0-6714-11e7-81f4-42018031c774.png)

**Comment by mckaygerhard** [comment]
this issue was solved, closed with pull #36 merged
